playbooks/ci_system_info.py

```diff
@@ -0,0 +1,43 @@
+import platform
+import textwrap
+import subprocess
+import psutil
+
+def write_system_info(adoc_file):
+
+    system_template = textwrap.dedent(
+            """
+            kernel version: {_kernel_version_}
+            boot options: {_boot_options_}
+            CPU architecture: {_cpu_arch_}
+            Number of logic CPUs : {_cpu_log_} 
+            RAM: {_ram_}
+            Network interfaces: 
+            """
+            )
+
+    network_template = textwrap.dedent(
+            """
+            {_net_if_name_}
+            """
+    )
+
+    adoc_file.write(
+            system_template.format(
+                _kernel_version_ = f"{platform.platform()} {platform.version()}",
+                _boot_options_ = subprocess.run(['cat', '/proc/cmdline'], capture_output=True, text=True).stdout.strip(),
+                _cpu_arch_ = platform.machine(),
+                _cpu_log_ = psutil.cpu_count(),
+                _ram_ = str(round(psutil.virtual_memory().total / (1024.0 **3)))+" GB" 
+            )
+    )
+
+    for i in range(len(psutil.net_if_stats())):
+        adoc_file.write(
+            network_template.format(
+                _net_if_name_ = list(psutil.net_if_stats())[i],
```

Comment:
That display all the interfaces including loopback, docker or internal bridges.
It would be better to display only physical interfaces.
It would be even better if you can display what network card it is and what capabilities it have.